fastlane/lib/fastlane/actions/upload_symbols_to_crashlytics.rb

```diff
@@ -94,7 +94,9 @@ def self.details
         [
           "This action allows you to upload symbolication files to Crashlytics.",
           "It's extra useful if you use it to download the latest dSYM files from Apple when you",
-          "use Bitcode"
+          "use Bitcode. This action will not fail the build if one of the uploads failed.",
+          "The reason for that is that sometimes some of dSYM files are invalid, and we don't want",
```

Comment:
I'm not sure if this is tvOS specific, but the dSYM files I download for tvOS apps do not have a file extension, they download as a single file named `dSYMs`, which is actually a zipped archive. I have to manually add `.zip` to them to get Crashlytics to accept them when I upload through the site.